company-profiles/two.md

```diff
@@ -0,0 +1,31 @@
+# Two
+
+## Company blurb
+
+Two is an Artificial Reality company lead by Pranav Mistry (creator of SixthSense, the world’s first wearable computer) and a team of world-class technologists, Two is creating a new category of immersive AI experiences that are more natural, more intuitive, and more expressive than text and voice.
+
+## Company size 
+
+Started less than 1 year ago in Silicon Valley.
+
+## Remote status
+
+There is no information on how the remote culture works at Skillshare. As of October 2020 there are two remote job positions available.
```

Comment:
This is outdated and/or incorrect as it mentions Skillshare.

Reply:
I apologize for the inconvenience, I have raised a new pull request as the head of the repository was deleted by me. I hope you'll review it soon and find it helpful.

Thanks you so much for considering my contribution 🙏